lib/config-win32.h

```diff
@@ -431,49 +448,87 @@ Vista
 /*                           LDAP SUPPORT                           */
 /* ---------------------------------------------------------------- */
 
-#if defined(CURL_HAS_NOVELL_LDAPSDK)
+#ifdef CURL_HAS_NOVELL_LDAPSDK
 #undef USE_WIN32_LDAP
 #define HAVE_LDAP_SSL_H 1
 #define HAVE_LDAP_URL_PARSE 1
 #elif defined(CURL_HAS_OPENLDAP_LDAPSDK)
 #undef USE_WIN32_LDAP
 #define HAVE_LDAP_URL_PARSE 1
-#else
+#elif !defined(CURL_WINDOWS_UWP) && !defined(UNDER_CE)
```

Comment:
lib/config-win32.h had `USE_WIN32_LDAP` enabled, just without the `HAVE_LDAP_SSL`. Is the behavior change on purpose?

Reply:
Well spotted! It did indeed, but it also had `#define CURL_DISABLE_LDAP 1` later in the file, so LDAP was left unused by `config-win32ce.h`.

LDAP is also broken/missing from CeGCC. As for MSVC, my botched and non-standard MSVC 2008 toolchain for WinCE has both `wldap32` implib and `winldap.h` header, but no `subauth.h`, required by `lib/ldap.c`. This may not be a 100% indicator.

Internet search gives mixed signals. It may have dependended on WinCE version or other factors.

I went back and forth with its status for non-CeCC, so definitive info would be nice.